presto-product-tests/src/main/java/com/facebook/presto/tests/AlterTableTests.java

```diff
@@ -34,67 +33,88 @@
+            assertThat(query(format("ALTER TABLE %s RENAME COLUMN n_nationkey TO nationkey", tableName), UPDATE))
+                    .hasRowsCount(1);
+            assertThat(query(format("SELECT count(nationkey) FROM %s", tableName)))
+                    .containsExactly(row(25));
+            assertThat(() -> query(format("ALTER TABLE %s RENAME COLUMN nationkey TO nATIoNkEy", tableName)))
+                    .failsWithMessage("Column 'nationkey' already exists");
+            assertThat(() -> query(format("ALTER TABLE %s RENAME COLUMN nationkey TO n_regionkeY", tableName)))
+                    .failsWithMessage("Column 'n_regionkey' already exists");
 
-        assertThat(query(format("ALTER TABLE %s RENAME COLUMN nationkey TO n_nationkey", TABLE_NAME)));
+            assertThat(query(format("ALTER TABLE %s RENAME COLUMN nationkey TO n_nationkey", tableName)));
+        }
+        finally {
+            dropTestTable(tableName);
+        }
     }
 
     @Test(groups = {ALTER_TABLE, SMOKE})
     public void addColumn()
     {
-        query(format("CREATE TABLE %s AS SELECT * FROM nation", TABLE_NAME));
+        String tableName = uniqueTableName();
+        try {
+            query(format("CREATE TABLE %s AS SELECT * FROM nation", tableName));
 
-        assertThat(query(format("SELECT count(1) FROM %s", TABLE_NAME)))
-                .containsExactly(row(25));
-        assertThat(query(format("ALTER TABLE %s ADD COLUMN some_new_column BIGINT", TABLE_NAME)))
-                .hasRowsCount(1);
-        assertThat(() -> query(format("ALTER TABLE %s ADD COLUMN n_nationkey BIGINT", TABLE_NAME)))
-                .failsWithMessage("Column 'n_nationkey' already exists");
-        assertThat(() -> query(format("ALTER TABLE %s ADD COLUMN n_naTioNkEy BIGINT", TABLE_NAME)))
-                .failsWithMessage("Column 'n_naTioNkEy' already exists");
+            assertThat(query(format("SELECT count(1) FROM %s", tableName)))
+                    .containsExactly(row(25));
+            assertThat(query(format("ALTER TABLE %s ADD COLUMN some_new_column BIGINT", tableName)))
+                    .hasRowsCount(1);
+            assertThat(() -> query(format("ALTER TABLE %s ADD COLUMN n_nationkey BIGINT", tableName)))
+                    .failsWithMessage("Column 'n_nationkey' already exists");
+            assertThat(() -> query(format("ALTER TABLE %s ADD COLUMN n_naTioNkEy BIGINT", tableName)))
+                    .failsWithMessage("Column 'n_naTioNkEy' already exists");
+        }
+        finally {
+            dropTestTable(tableName);
+        }
+    }
+
+    private void dropTestTable(String tableName)
+    {
+        try {
+            query(format("DROP TABLE IF EXISTS %s", tableName));
+        }
+        catch (Exception e) {
+            Logger.get(getClass()).warn(e, "failed to drop table: %s", tableName);
+        }
+    }
+
+    private static String uniqueTableName()
```

Comment:
Create the `Tables` utility class in this commit so that we don't have to move it on the next commit.